llama_stack/core/stack.py

```diff
@@ -326,13 +326,19 @@ async def initialize(self):
 
         dist_registry, _ = await create_dist_registry(self.run_config.metadata_store, self.run_config.image_name)
         policy = self.run_config.server.auth.access_policy if self.run_config.server.auth else []
+
+        # Add internal implementations before resolving other providers so they can depend on them
```

Comment:
i expect the inclusion of Api.prompts in the dep list in registry/agents.py to take care of this.

what's the error you get when you don't include this change?